builder/llpcBuilderImplInOut.cpp

```diff
@@ -730,6 +763,15 @@ Value* BuilderImplInOut::ReadBuiltIn(
         LLPC_ASSERT(pIndex == nullptr);
         args.push_back((pVertexIndex != nullptr) ? pVertexIndex : getInt32(InvalidValue));
         break;
+    case ShaderStageFragment:
```

Comment:
pVertexIndex has different meaning. This is error-prone. Please add comments to the function parameter mentioning this special meanings.